hugo/layouts/report.html

```diff
@@ -198,25 +198,50 @@ <h2 class="mb-3">About Your Home</h2>
                   "card" $card
                   "Params" $.Params
                   "is_multicard" true
-                  "special_case_multi_card" $.Params.special_case_multi_card
+                  "special_case_multi_card" false
                 )
               }}
             {{ end }}
           </div>
           {{ end }}
         </div>
 
+        {{/*
+          Use the global store for frankencard assignment so that we can update
+          it within the scope of conditional branches and loops below
+        */}}
+        {{ $.Store.Set "frankencard" false }}
+
         {{ if .Params.special_case_multi_card }}
-          {{ $card := index .Params.cards 0 }}
-          {{
-            template "card-comps"
-            (dict
-              "card" $card
-              "Params" $.Params
-              "is_multicard" true
-              "special_case_multi_card" $.Params.special_case_multi_card
-            )
-          }}
+          {{/*
+            Extract frankencard to use as base for comps and values in the
+            small recent multicard case
+          */}}
+          {{ range .Params.cards }}
+            {{ if .is_frankencard }}
+              {{ $.Store.Set "frankencard" . }}
+            {{ end }}
+          {{ end }}
+
+          {{ $frankencard := $.Store.Get "frankencard" }}
+
+          {{ if not $frankencard }}
+            {{
+              errorf
+              "No card with is_frankencard == true found in .Params.cards for small multicard PIN %q"
```

Comment:
We also add a data test in https://github.com/ccao-data/data-architecture/pull/864 to ensure there is always a `true` value for at least one card in all small multicard PINs, but this error message will give us an additional layer of defensiveness to guard against the generation of reports that are accidentally missing information.